manifests/init.pp

```diff
@@ -45,6 +45,12 @@
   Array[Cron::User]    $users_deny         = [],
   Boolean              $manage_users_allow = false,
   Boolean              $manage_users_deny  = false,
+  Boolean              $manage_crontab     = false,
```

Comment:
can you please add puppet-strings documentation for all new parameters?

Reply:
puppet-strings docs added